src/main/cliapp/src/containers/diseaseAnnotationsPage/DiseaseAnnotationsTable.js

```diff
@@ -315,6 +315,8 @@ export const DiseaseAnnotationsTable = () => {
         let annotations = [...diseaseAnnotations];
         annotations[event.index].subject = data.data.entity.subject;
         annotations[event.index].object = data.data.entity.object;
+        annotations[event.index].dateLastModified = data.data.entity.dateLastModified;
+        annotations[event.index].modifiedBy = data.data.entity.modifiedBy;
```

Comment:
@kthorat-prog then with this code shouldn't we just be doing a:
`annotations[event.index] = data.data.entity;`

Reply:
@oblodgett Exactly what I thought but I can't recall why only specifically we did it for subject & object only. Or is it that after adding new fields, they are left out?

I would want to do it in just one line though.